pkg/cri/server/image_pull.go

```diff
@@ -154,6 +159,31 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 		tracing.Attribute("image.ref", ref),
 		tracing.Attribute("snapshotter.name", snapshotter),
 	)
+
+	// Pull image using transfer service
+	if c.config.PluginConfig.ImagePullWithTransferService {
+
+		var sopts []transferimage.StoreOpt
+
+		sopts = append(sopts, transferimage.WithUnpack(platforms.DefaultSpec(), snapshotter))
```

Comment:
Also try passing in `image.WithPlatforms(platforms.DefaultSpec()` to avoid pulling multiple platforms

Reply:
Thanks for reviewing! Fixed.